src/renderer/components/ft-list-video/ft-list-video.js

```diff
@@ -367,6 +376,13 @@ export default defineComponent({
       }
     },
 
+    displayDuration: function () {
+      if (this.useDeArrowTitles && (this.duration === '' || this.duration === '0:00') && this.deArrowCache?.videoDuration) {
+        return formatDurationAsTimestamp(this.deArrowCache?.videoDuration)
```

Comment:
You can remove the optional chaining operator here, as you are already checking that in the surrounding `if` statement, so inside the `if` statement it's confirmed to exist.
```suggestion
        return formatDurationAsTimestamp(this.deArrowCache.videoDuration)
```